Source/i_sound.c

```diff
@@ -740,17 +740,94 @@ void I_InitSound(void)
       // haleyjd 04/11/03: don't use music if sfx aren't init'd
       // (may be dependent, docs are unclear)
       if(!nomusicparm)
-         I_InitMusic();
+      {
```

Comment:
Don't we have to initialize SDL_Mixer even in the `nomusic` case for the SFX to work?

Reply:
We initialize it earlier in this function. I have tested that `-nosfx` and `-nomusic` now work correctly, at least on Windows.

Reply:
👍

Reply:
👍